promscale/tobs/tobs.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Install the observability stack (tobs) for Kubernetes
+excerpt: Install tobs
+product: promscale
+keywords: [analytics, Kubernetes]
+tags: [install, k8s]
+related_pages:
+  - /promscale/:currentVersion:/recommendations/resource-recomm
+  - /promscale/:currentVersion:/send-data/
+---
+
+import PromscaleConfigurationRecommendations from 'versionContent/_partials/_promscale-configuration-recommendations.mdx';
+
+# Install the observability stack (tobs) for Kubernetes
+The observability stack (tobs), is a pre-packaged distribution of observability
+tools and dashboard interfaces which can be installed on any existing Kubernetes
+cluster. It includes many of the most popular open-source observability tools
+including Prometheus, Grafana, Jaeager, TimescaleDB, and [other components][stack-components].
+Together, these provide a maintainable solution to analyze the traffic on the
+server and identify any potential problems with a deployment. You can use Helm
```

Comment:
Help me understand better... I always thought that we use metrics and traces to analyze this data to identify whether there’s a user-impacting problem and if so – what the root cause is to fix it fast. That is why it was rewritten to avoid the jargon metrics and traces.

Reply:
Firstly the dashboards offered by Tobs offer visibility into the systems. You are correct on this part. But Tobs doesn't offer the solution or fix for the cause. 

I'm not okay with this sentence because:
1. `analyze the traffic on the server`: This doesn't explain anything and when we say generic word server. Which server are we referring to? Kubernetes api-server? Prometheus server? and server as the term is common for everything based on ones understanding.

Reason for my suggestion:
1. `to observe deployments in Kubernetes cluster using both metrics and traces`: We observe deployments in K8s cluster primarily all components that are packaged with Tobs. If the user wants Tobs to observe other deployments the user has to configure metrics endpoints in Prometheus config and Otel collector endpoint on their applications end. 

You can either change this to the one I suggested or to `observe deployments in Kubernetes cluster to identify any potential problems using both metrics and traces`